include/RAJA/Iterators.hxx

```diff
@@ -0,0 +1,185 @@
+#ifndef RAJA_ITERATORS_HXX
```

Comment:
@trws Please add the release statement to the top of this new file -- can be cut and pasted from some other file.  Thanks.